src/transformers/utils/quantization_config.py

```diff
@@ -567,9 +577,7 @@ def post_init(self):
             )
 
         if self.version not in [AWQLinearVersion.GEMM, AWQLinearVersion.GEMV]:
-            raise ValueError(
-                f"Only supported versions are in [AWQLinearVersion.GEMM, AWQLinearVersion.GEMV] - not recognized version {self.version}"
-            )
+            self.version = AWQLinearVersion.from_str(self.version)
```

Comment:
Other change to be inline with above (I can't make a suggestion across additions and deletions in the diff)
```suggestion
```